RCTAppleHealthKit/RCTAppleHealthKit+TypesAndPermissions.m

```diff
@@ -64,10 +64,84 @@ - (nullable HKObjectType *)getReadPermFromText:(nonnull NSString*)key {
+    } else if ([@"Niacin" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryNiacin];
+    } else if ([@"PantothenicAcid" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryPantothenicAcid];
+    } else if ([@"Phosphorus" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryPhosphorus];
+    } else if ([@"Potassium" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryPotassium];
+    } else if ([@"Protein" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryProtein];
+    } else if ([@"Riboflavin" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryRiboflavin];
+    } else if ([@"Selenium" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietarySelenium];
+    } else if ([@"Sodium" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietarySodium];
+    } else if ([@"Sugar" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietarySugar];
+    } else if ([@"Thiamin" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryThiamin];
+    } else if ([@"VitaminA" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryVitaminA];
+    } else if ([@"VitaminB12" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryVitaminB12];
+    } else if ([@"VitaminB6" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryVitaminB6];
+    } else if ([@"VitaminC" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryVitaminC];
+    } else if ([@"VitaminD" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryVitaminD];
+    } else if ([@"VitaminE" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryVitaminE];
+    } else if ([@"VitaminK" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryVitaminK];
+    } else if ([@"Zinc" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryZinc];
+    } else if ([@"Water" isEqualToString:key]) {
+        return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierDietaryWater];
+    } else if ([@"BloodGlucose" isEqualToString:key]) {
```

Comment:
We already have the `BloodGlucose` at line 171. We can remove that one and stick with the new entry you've added, as that keeps it more organized

What do you think?

Reply:
@lucaspbordignon I agree. Can you assist m, though? 😅 

When I visit my repository @ this branch, I have: 

> This branch is 1 commit ahead, 28 commits behind agencyenterprise:master.

Should I update the branch first and then make this change or directly push my changes to my branch? 

Thanks in advance!

Reply:
You could push your changes to your branch

Reply:
As @Macelai stated, you could just push to your branch and it will update the PR (as you already did  🙂)